pkg/api/validation/validation.go

```diff
@@ -2844,10 +2799,16 @@ func ValidateNodeUpdate(node, oldNode *api.Node) field.ErrorList {
 	// Clear status
 	oldNode.Status = node.Status
 
+	// update taints
```

Comment:
But you said PodSpec is already mutable (for two fields). :)

Do you disagree that handling the scenarios I describe need some kind of ability to update tolerations on existing pods?

I'm trying to understand if your objection is "we have too much stuff that relies on this assumption right now and not enough time to fix it" or "adding tolerations can never be safe." If the latter, we should understand the issues before moving taints/tolerations feature to beta, as the feature really relies on being able to add tolerations to existing pods.